src/main/java/org/java_websocket/exceptions/InvalidDataException.java

```diff
@@ -60,6 +73,19 @@ public InvalidDataException(int closecode, String s) {
     this.closecode = closecode;
   }
```

Comment:
This constructor should initialize httpErrorCode = null

Reply:
It's not required as I used Integer so if not initialized it will be null